crates/weaver_forge/src/formats/html.rs

```diff
@@ -550,9 +552,9 @@ If a specific domain defines its own set of error identifiers (such as HTTP or g
 it's RECOMMENDED to:
 <ul>
   <li>Use a domain-specific attribute
-  <li>Set {@code error.type} to capture all errors, regardless of whether they are defined within the domain-specific set or not.
+  <li>Set {@code error.type} to capture all errors, regardless of whether they are defined within the domain-specific set or not
 </ul>
-"##
+And something more."##
```

Comment:
it was confusing for me at first why it sometimes ended with a newline and sometimes not, which was part of the motivation for #509